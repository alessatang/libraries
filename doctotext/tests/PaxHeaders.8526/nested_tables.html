29 mtime=1382706428.32303706
30 atime=1389151481.611852358
30 ctime=1389151481.611852358
