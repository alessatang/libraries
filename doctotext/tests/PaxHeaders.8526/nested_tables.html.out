30 mtime=1382706428.324037061
30 atime=1389151481.612852354
30 ctime=1389151481.612852354
